### DISCUSSION TITLE
Media feed for each List?

### DISCUSSION THREAD

**Discussion by Luke-L** [robertodoering/harpy#424] [Ideas]
Hands down my favorite feature of Harpy is the fullsize media feed. It sets this app apart from all the other twitter clients I've tried. However, this media feed only works for your twitter homepage, and is not available when browsing lists. 
Is there any chance this will be added soon, along with the media filters that are already in the homepage?

**Comment by robertodoering** [comment]
Hey @Luke-L
Thanks for the suggestion!
It definitely makes sense to have the media timeline available for lists as well.

I have created an issue for it: https://github.com/robertodoering/harpy/issues/425

Regarding filters for lists, that is also already planned. See: https://github.com/robertodoering/harpy/issues/352

**Comment by zarbout** [comment]
Hi Community. Have I missed a way to get tweets sorted by Top tweets (versus timeline) or it is not possible for now ? I'd like to have under a given list and not only home. Thank you

**Reply by robertodoering** [reply]
Tweets are always sorted chronologically.
Twitter doesn't provide 3rd party apps like harpy to implement their algorithmic sorting and any client side solution would never work great so this is what we are stuck with.

**Reply by zarbout** [reply]
Crystal clear! Thank you and keep it up.
